packages/astro/src/cli/add/index.ts

```diff
@@ -361,7 +361,19 @@ export async function add(names: string[], { flags }: AddOptions) {
 					} to your project:\n${list}`,
 				),
 			);
-			logger.info('SKIP_FORMAT', msg.success("Import './src/styles/global.css' in a layout"));
+			if (integrations.find((integration) => integration.integrationName === 'tailwind')) {
+				const code = boxen(
+					getDiffContent('---\n---', "---\nimport './src/styles/global.css'\n---")!,
+					{
+						margin: 0.5,
+						padding: 0.5,
+						borderStyle: 'round',
+						title: 'src/layouts/Layout.astro',
+					},
+				);
+				logger.warn('SKIP_FORMAT', msg.actionRequired('You must add the following in a layout:\n'));
```

Comment:
```suggestion
				logger.warn('SKIP_FORMAT', msg.actionRequired('You must import your Tailwind stylesheet, e.g.:\n'));
```
I like how short yours is, but is it *always*, *literally*, *exactly* accurate?  Is it always "copy this exact code in this exact spot?"

Non-blocking, but just a note that you ARE going to get people typing exactly, literally that if you phrase it that way. So be sure that's what you want!

Reply:
I believe they can have multiple layout files that would need to be updated. Or even a file that has a different name, like `src/layouts/MySiteLayout.astro`. 

This is at least what `npx create-astro@latest` will generate.
<img width="1469" alt="image" src="https://github.com/user-attachments/assets/918aff1f-4f1e-4202-a30d-8c1d320c9ce3" />

Not sure how to handle this best but figure this might give you some ideas?

Reply:
@sarah11918 I think your suggestion is great, I would add something like this
```diff
-You must import your Tailwind stylesheet, e.g.:
+You must import your Tailwind stylesheet, e.g. in a shared layout:
```

Reply:
Used the latest copy text from @florian-lefebvre in 98d303b9ab0e! I believe this is resolved now and happy to make this better later if there is user confusion.

Reply:
Lovely job, Florian! I was trying to keep it as short as you had it. Might be worth checking an updated screenshot to make sure it doesn't look too ugly? If I got to pick based on wording alone, absolutely. Just make sure it doesn't cause a visual mess!